addons/binding/org.openhab.binding.onkyo/ESH-INF/config/config.xml

```diff
@@ -29,10 +29,16 @@
 		</parameter>
 		<parameter name="volumeLimit" type="integer" min="0" max="100">
 			<label>Volume Limit</label>
-			<description>Limit maximum volume level to defined value.</description>
+			<description>Limit maximum volume level to defined percentage.</description>
 			<default>100</default>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="volumeScale" type="integer" min="0" max="100">
+            <label>Volume Scale</label>
+            <description>Select volume scale (0 = 0-100, 1 = 0-100 in 0.5 steps, 2 = 0-80, 3 = 0-50)</description>
+            <default>0</default>
+            <advanced>true</advanced>
```

Comment:
Maybe add a fixed options list here to make sure the parameter only accepts one of the four expected values.

Reply:
Done -> coefficient is now exposed directly as this provides more flexibility